app/models/host/managed.rb

```diff
@@ -569,7 +569,7 @@ def inherited_attributes
       inherited_attrs.concat(%w{operatingsystem_id architecture_id compute_resource_id})
       inherited_attrs << "subnet_id" unless compute_provides?(:ip)
       inherited_attrs << "subnet6_id" unless compute_provides?(:ip6)
-      inherited_attrs.concat(%w{medium_id ptable_id pxe_loader}) if pxe_build?
+      inherited_attrs.concat(%w{medium_id ptable_id pxe_loader}) unless host.image_build?
```

Comment:
I believe this needs to me `image_build`, please drop the `host`.

Reply:
Damn I thought I found them all. Good catch.